fairscale/optim/oss.py

```diff
@@ -464,14 +469,14 @@ def rank_local_state_dict(rank: int, state_dict: dict) -> dict:
         param_groups = state_dict["param_groups"][state_dict["partition"][rank][0] : state_dict["partition"][rank][1]]
         return {"state": state_dict["state"][rank], "param_groups": param_groups}
 
-    def load_local_state_dict(self, state_dict: dict) -> None:
```

Comment:
I'm not sure whether this is really needed, it was mostly convenient before since the saved state was a list of sharded states, now that it's not the case I figured that was not needed anymore